server/task_check_scheduler.go

```diff
@@ -404,7 +404,9 @@ func (s *Server) passCheck(ctx context.Context, server *Server, task *api.Task,
 		return false, err
 	}
 	for _, result := range checkResult.ResultList {
-		if result.Status == api.TaskCheckStatusError || result.Status == api.TaskCheckStatusWarn {
+		if result.Status == api.TaskCheckStatusError ||
+			// Skip empty schema review policy warning cause it is only used to remind the user to use schema system.
+			(result.Status == api.TaskCheckStatusWarn && result.Code != common.TaskCheckEmptySchemaReviewPolicy) {
```

Comment:
```suggestion
		if result.Status == api.TaskCheckStatusError {
```
IMO, we should only show warns in the web UI, and return check success here.
WDYT? @tianzhou @d-bytebase